lib/jtree.browser.js

```diff
@@ -0,0 +1,6126 @@
+"use strict"
```

Comment:
Can you add a comment here explaining that we've copy pasted this from this library and add a link to the library and this specific file in their source control?

Reply:
Yes good idea. I can do it after pull